File: my-shop-selenium/src/main/java/data/dto/UserDTO.java
package data.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDTO {
    private String email;
    private String password;
    private String fistName;
    private String lastName;
    private String dayOfBirth;
    private String monthOfBirth;
    private String yearOfBirth;
    private boolean isMen;
}

File: my-shop-cypress/cypress/e2e/data/dto/user-dto.ts
export default class UserDTO {
    email: string;
    password: string;
    fistName: string;
    lastName: string;
    dayOfBirth: string;
    monthOfBirth: string;
    yearOfBirth: string;
    isMen: boolean;

    constructor(email: string, password: string, fistName: string, lastName: string, dayOfBirth: string, monthOfBirth: string, yearOfBirth: string, isMen: boolean) {
        this.email = email;
        this.password = password;
        this.fistName = fistName;
        this.lastName = lastName;
        this.dayOfBirth = dayOfBirth;
        this.monthOfBirth = monthOfBirth;
        this.yearOfBirth = yearOfBirth;
        this.isMen = isMen;
    }
}